tests/test_bs.py

```diff
@@ -396,7 +396,8 @@ def test_transaction(load_config, setup_clients):
 
     api = 'v1/nonce'
     data = '{"address":"1"}'
-    p = subprocess.Popen(['./kubectl-cmd.sh', '%s' % client_ip, "%s" % data, api], stdin=subprocess.PIPE,
+    namespace = testconfig['namespace']
```

Comment:
why can't we just use `testconfig['namespace']` as the last param?

Reply:
Just for readability.....

Reply:
IMO there's no big difference in readability

Reply:
Fixed